mysql-test/suite/rpl/t/rpl_bug-lp-1667307.cnf

```diff
@@ -0,0 +1,30 @@
+!include ../my.cnf
+
+[mysqld.1]
+enforce-gtid-consistency=ON
+gtid-mode=ON
+master-info-repository=TABLE
```

Comment:
No, we need it, see topology in the test source code:

```
--let $rpl_topology= 1->3,3->2,1->2
--let $rpl_multi_source= 1
--let $use_gtids= 1
--source include/rpl_init.inc
```

For such topology we need multi-source replication, what requires master and relay log info format to be "TABLE". I am not sure if it is possible to have some servers with "multisource" option on, and some of them with the option switched off in replication graph, but, anyway, "rpl_init.inc" does not allow this, if "$rpl_multi_source" is on, then it's on for all servers pointed in the graph.